content/iter8 trend/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2020-08-11T12:00:00+00:00
+menuTitle: iter8-trend
+title: Getting Started with iter8-trend
+weight: 25
+summary: Describes how to install and use iter8-trend
+---
+
+iter8-trend collects metrics of past experiments and summarizes them to show
+trends that could expose performance problems creeping up over time, which might
+not be obvious if one only compares consecutive versions during canary testing.
+This is invaluable to developers when tracking down a performance problem and
+pin-pointing it to a particular version that was deployed in the past.
+
+This is an optional component to [iter8](http://github.com/iter8-tools) and
+cannot run standalone. It should be installed either as part of iter8
+installation process or separately after iter8 is installed.
+
+The following short video introduces iter8-trend:
+
+[![Short Video](https://img.youtube.com/vi/FOtyqJPMj14/hqdefault.jpg)](https://youtu.be/FOtyqJPMj14)
```

Comment:
@kalantar @huang195 Can someone replace this using Hugo's Youtube shortcode? 

See here: https://gohugo.io/content-management/shortcodes/#youtube

Reply:
done